services/auth/src/main/java/org/eclipse/hono/service/auth/impl/ApplicationConfig.java

```diff
@@ -93,16 +94,27 @@ public ObjectFactoryCreatingFactoryBean authServerFactory() {
     }
 
     /**
-     * Exposes properties for configuring the application properties a Spring bean.
+     * Exposes properties for configuring the application properties as a Spring bean.
      *
      * @return The application configuration properties.
      */
     @Bean
     @ConfigurationProperties(prefix = "hono.app")
-    public ApplicationConfigProperties applicationConfigProperties(){
+    public ApplicationConfigProperties applicationConfigProperties() {
         return new ApplicationConfigProperties();
     }
 
+    /**
+     * Exposes properties for configuring the health check as a Spring bean.
+     *
+     * @return The health check configuration properties.
+     */
+    @Bean
+    @ConfigurationProperties(prefix = "hono.health-check")
+    public ServerConfig healthCheckConfigProperties() {
```

Comment:
@sophokles73 I am unsure if we want to name it that way even though we create a `ServerConfig` object. WDYT?

Reply:
That's ok FMPOV